tests/rptest/tests/audit_log_test.py

```diff
@@ -554,7 +554,6 @@ def number_of_records_matching(filter_by, n_expected):
         self.logger.debug("Finished 500 api calls with management disabled")
         _ = number_of_records_matching(api_keys, 1000)
 
-    @ok_to_fail  # https://github.com/redpanda-data/redpanda/issues/14565
```

Comment:
Don't forget to add a `fixes` line in the github commit message. GH will automatically pick this up and link the commit to the relevent issue.

Like this:
```
- Fixes: #<issue number> 
```

Reply:
Heh, I've been manually adding it to the PR header...commit msg makes sense for single-commit diffs though 🙂